swagger.json

```diff
@@ -959,6 +959,203 @@
+        "/fleet/unassigned_driving_segments": {
+            "get": {
+                "tags": [
+                    "Fleet", "Default"
+                ],
+                "summary": "/fleet/unassigned_driving_segments",
+                "description": "Get the unassigned driving segments for a specified range.",
+                "operationId": "get_fleet_unassigned_driving_segments",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/accessTokenParam"
+                    },
+                    {
+                        "name": "unassignedDrivingSegmentsParam",
+                        "required": true,
+                        "in": "body",
+                        "schema": {
+                            "type": "object",
+                            "required": [
+                                "startMs"
+                            ],
+                            "properties": {
+                                "startMs": {
+                                    "type": "integer",
+                                    "description": "Beginning of the time range, specified in milliseconds UNIX time.",
+                                    "example": 1462878398034
+                                },
+                                "endMs": {
+                                    "type": "integer",
+                                    "description": "End of the time range, specified in milliseconds UNIX time.",
+                                    "example": 1462881998034
+                                }
+                            }
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Unassigned driving segments for duration.",
+                        "schema": {
+                            "type": "object",
+                            "properties": {
+                                "unassigned_driving_segments": {
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/definitions/UnassignedDrivingSegment"
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/fleet/unassigned_driving_segments/annotate": {
+            "post": {
+                "tags": [
+                    "Fleet", "Default"
+                ],
+                "summary": "/fleet/unassigned_driving_segments/annotate",
+                "description": "Annotate an unassigned driving segment.",
```

Comment:
Are we requiring only one of `start_ms` or `created_at_ms`? Let's add to the description what the options are for identifying the segment to annotate.